### PR TITLE
[new locale] ar-ps: Arabic (Palestine)

### PR DESCRIPTION
**Pull request by majdal**
Hello - Just adding a new local that has Syriac month names, used in Palestine, Lebanon, Syria, Jordan, and Iraq ([see](https://en.wikipedia.org/wiki/Assyrian_calendar)).

### CODE REVIEW

**Comment by ichernev** [comment]
@majdal  when adding a new locale you need to put it under `src/locale/xx.js` and add tests to `src/test/locale/xx.js`. Files under `locale/*` are autogenerated and not touched by hand.

**Comment by marwahaha** [comment]
Looks like the prettier check failed. I think you can fix it with `npm run prettier-fmt`

**Comment by marwahaha** [comment]
Unfortunately, the months are not parsing correctly! This is why the tests are failing.

File: src/locale/ar-ps.js [hunk 1]
```diff
@@ -85,9 +85,18 @@ export default moment.defineLocale('ar-ps', {
...
     preparse: function (string) {
         return string
-            .replace(/[١٢٣٤٥٦٧٨٩٠]/g, function (match) {
+            .replace(/[٣٤٥٦٧٨٩٠]/g, function (match) {
                 return numberMap[match];
             })
+            .split('') // reversed since negative lookbehind not supported everywhere
+            .reverse()
+            .join('')
+            .replace(/[١٢](?![\u062a\u0643])/g, function (match) {
```
Comment (marwahaha) [inline src/locale/ar-ps.js:94]: This is done to avoid parsing the month numbers used in `monthsShort`.

**Comment by majdal** [comment]
I'll need to change the tests too: we should be using Arabic numerals (1, 2, 3...) instead of the currently defined Indian numerals (١، ٢، ٣...).

**Comment by marwahaha** [comment]
Ok @majdal . Let me know how I can assist!

**Comment by marwahaha** [comment]
@majdal any updates?

**Comment by ichernev** [comment]
Merged in 794938ae326b7b38ea9921a7b5c84e773794a6ed

**Pull request closed without merging**